src/integration/java/io/pinecone/integration/controlPlane/pod/ConfigureIndexTest.java

```diff
@@ -50,58 +54,51 @@ public void configureIndexExceedingQuota() {
         ConfigureIndexRequestSpec spec = new ConfigureIndexRequestSpec().pod(pod);
         ConfigureIndexRequest configureIndexRequest = new ConfigureIndexRequest().spec(spec);
         try {
-            isIndexReady(indexName, controlPlaneClient);
             controlPlaneClient.configureIndex(indexName, configureIndexRequest);
         } catch (PineconeForbiddenException forbiddenException) {
             assert (forbiddenException.getLocalizedMessage().contains("Increase your quota or upgrade to create more indexes."));
```

Comment:
In all these cases where we're *expecting* a failure, we need to make sure that the test fails if the exception is not thrown. Here's a general example:

```java
try {
    // make the call that should result in an exception 
    controlPlanClient.configureIndex(indexName, configureIndexRequest);

    // call Assert.fail if we get to this point the code should have thrown,
    // but didn't, so we need to fail the test.
    fail("Should have throw PineconeForbiddenException");

} (catch PineconeForbiddenException expected) { 
    // I like naming the exception as "expected" to make it obvious it's the test success path

    // Do nothing.

    // (Or possibly validate the contents of the exception, if it matters to the client.) 
    // But in this case, do we actually care if the string error message changes? Probably not
}
```

Reply:
Alternatively, you can wrap the call in `Assert.assertThrows` like:

```java
assertThrows(PineconeForbiddenException.class,
     controlPlanClient.configureIndex(indexName, configureIndexRequest));
```

But that's a little less flexible especially when you'd like to do some validation on the exception itself.